### PR TITLE
TGU - Kneejerk reaction, fixed smoking, gamemode is GONE.

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

Updates us to latest TG yet again
Going to test it soon, but AFAIK nothing should break. Also this needs host action due to db
Aaalso after merging this you can enable mapdiffbot

### CODE REVIEW

File: config/config.txt [hunk 1]
```diff
@@ -3,7 +3,6 @@
 $include game_options.txt
 $include dbconfig.txt
 $include comms.txt
-$include antag_rep.txt
```
Comment (JohnFulpWillard) [inline config/config.txt:6]: Finally...

File: code/modules/admin/sql_ban_system.dm [hunk 2]
```diff
@@ -300,10 +300,6 @@
 				ROLE_BROTHER,
 				ROLE_CHANGELING,
 				ROLE_CULTIST,
-				/// Fulp edit - Bloodsuckers
```
Comment (JohnFulpWillard) [inline code/modules/admin/sql_ban_system.dm:303]: uh oh

**Comment by remove32** [comment]
well this has been TMed for several rounds, are we good?

**Comment by SgtHunk** [comment]
I think so. Got no bug reports related to fulp features. And my only mess-up (accidentally removing fulp antags from SQL bans) was fixed.

**Pull request merged**